### ISSUE TITLE
Twitter video download issue with cookies

### ISSUE THREAD

**Issue by open-mindt** [ytdl-org/youtube-dl#20855]
## Checklist

- [ ] I'm reporting a broken site support
- [x] I've verified that I'm running youtube-dl version **2019.04.24**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [ ] I've searched the bugtracker for similar issues including closed ones

## Verbose log

```
PS D:\> youtube-dl -v --cookies twitter.txt https://twitter.com/F1/status/1122028855936425985
[debug] System config: []
[debug] User config: []
[debug] Custom config: []
[debug] Command-line args: ['-v', '--cookies', 'twitter.txt', 'https://twitter.com/F1/status/1122028855936425985']
[debug] Encodings: locale cp949, fs utf-8, out utf-8, pref cp949
[debug] youtube-dl version 2019.04.24
[debug] Python version 3.7.3 (CPython) - Windows-10-10.0.18362-SP0
[debug] exe versions: ffmpeg N-93634-geeca67e023
[debug] Proxy map: {}
[twitter] 1122028855936425985: Downloading webpage
ERROR: Unable to extract OpenGraph title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
Traceback (most recent call last):
  File "c:\python\lib\site-packages\youtube_dl\YoutubeDL.py", line 796, in extract_info
    ie_result = ie.extract(url)
  File "c:\python\lib\site-packages\youtube_dl\extractor\common.py", line 529, in extract
    ie_result = self._real_extract(url)
  File "c:\python\lib\site-packages\youtube_dl\extractor\twitter.py", line 448, in _real_extract
    username = remove_end(self._og_search_title(webpage), ' on Twitter')
  File "c:\python\lib\site-packages\youtube_dl\extractor\common.py", line 1116, in _og_search_title
    return self._og_search_property('title', html, **kargs)
  File "c:\python\lib\site-packages\youtube_dl\extractor\common.py", line 1104, in _og_search_property
    escaped = self._search_regex(og_regexes, html, name, flags=re.DOTALL, **kargs)
  File "c:\python\lib\site-packages\youtube_dl\extractor\common.py", line 1004, in _search_regex
    raise RegexNotFoundError('Unable to extract %s' % _name)
youtube_dl.utils.RegexNotFoundError: Unable to extract OpenGraph title; please report this issue on https://yt-dl.org/bug . Make sure you are using the latest version; see  https://yt-dl.org/update  on how to update. Be sure to call youtube-dl with the --verbose flag and include its complete output.
```

## Description

WRITE DESCRIPTION HERE

When I tried to download a video of private account with "--cookies" option, error appeared.
Tried to download another video(not private)..  also got an error.
But remove "--cookies" option, download worked well..

**Comment by hackles** [comment]
I can confirm this issue.

**Comment by jmcbee** [comment]
I'm also experiencing this when I have the --cookies parameter in `~/.config/youtube-dl/config`.

**Comment by michealespinola** [comment]
I can confirm this issue.  As far as I can tell, the issue is specific to the `--cookies` option, and not specific to twitter site support.  This issue affects multiple video sites.

If it helps anyone else reading, @dstftw closed #22142 citing a duplicate of https://github.com/ytdl-org/youtube-dl/pull/15840

**Comment by remitamine** [comment]
try to test with the latest version and report back if this is still a problem.
